packages/story-editor/src/components/panels/design/border/borderWidth.js

```diff
@@ -41,70 +35,64 @@ import {
 import { MULTIPLE_DISPLAY_VALUE, MULTIPLE_VALUE } from '../../../../constants';
 import { DEFAULT_BORDER } from './shared';
 
-const INPUT_TOTAL_HEIGHT = 64;
-const INPUT_WIDTH = 44;
+const BorderInputsGrid = styled.div`
+  display: flex;
+  margin-bottom: 16px;
+`;
 
-const BorderRow = styled(Row)`
-  ${({ locked }) => locked && 'justify-content: normal'};
+const BorderInputsWrapper = styled.div`
+  margin-left: 1px;
+  ${({ locked }) => (locked ? 'width: calc(50% - 32px);' : '')}
 `;
 
-const StyledLockToggle = styled(LockToggle)`
-  ${focusStyle};
+const BorderWidthNumericInput = styled(NumericInput)`
+  border-radius: 0px;
+  margin-left: -1px;
+
+  ${({ locked, border }) =>
+    !locked &&
+    `
+    width: 25%;
+    svg {
+      width: 10px;
+      height: 10px;
+      margin-right: -14px;
+      ${border === 'right' ? 'transform: scaleX(-1);' : ''}
+      ${border === 'bottom' ? 'transform: scaleY(-1);' : ''}
+    }
+  `}
```

Comment:
@barklund  Not sure if this will still be applicable after refactoring this setup to a new component which receives the icon as a prop https://github.com/google/web-stories-wp/pull/9571/files#diff-2ed31d5dd240d240eab0a00cd85830c42410cd75d9bda8ddda97da7e689aa097R1